config-model/src/main/java/com/yahoo/vespa/model/application/validation/Validation.java

```diff
@@ -99,9 +105,17 @@
                 new ContainerRestartValidator(),
                 new NodeResourceChangeValidator()
         };
-        return Arrays.stream(validators)
-                .flatMap(v -> v.validate(currentModel, nextModel, overrides, now).stream())
-                .collect(toList());
+        List<ConfigChangeAction> actions = Arrays.stream(validators)
```

Comment:
Just to repeat my standard comment to your code: These stream statements that does everything are like a long sentence with lots of subsentences. It's easier to read shorter sentences, especially since it makes you name the thing they produce.

Reply:
But.
I.
Like.
Long.
Sentences.

I appreciate your comment on this; you're probably not the only one, so I will try to adjust for readability.